cmd/ore/aws/upload.go

```diff
@@ -236,6 +239,21 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	tagMap := make(map[string]string)
+	if len(uploadTags) > 0 {
+		for _, tag := range uploadTags {
+			splitTag := strings.SplitN(tag, "=", 2)
+			key, value := splitTag[0], splitTag[1]
```

Comment:
This will panic on bad input. Should either make this print and error and exit if `len(splitTag) != 2`